Makefile

```diff
@@ -30,7 +30,12 @@ clean:
 	rm -rf bin/
 
 run: build
-	sudo bin/server -i eth0
+	@if grep -q 'interface:' config.yaml; then \
+		INTERFACE=$$(grep 'interface:' config.yaml | awk '{print $$2}'); \
+	else \
+		INTERFACE=eth0; \
+	fi; \
+	sudo bin/server -c config.yaml -i $$INTERFACE
```

Comment:
why do you need this? We have defaults for all the flags. The interface should not be defined in the Makefile. If someone wants to change the interface during the development, they would do it in the config file.